src/js/actions/superselect.js

```diff
@@ -396,8 +396,17 @@ define(function (require, exports) {
 
                 if (!clickedSelectableLayerIDs.isEmpty()) {
                     // due to way hitTest works, the top z-order layer is the last one in the list
-                    var topLayerID = clickedSelectableLayerIDs.last(),
-                        topLayer = layerTree.byID(topLayerID),
+                    var topLayerID = clickedSelectableLayerIDs.findLast(function (layerID) {
+                        var layer = layerTree.byID(layerID);
+                        if (layerTree.minumumChildBounds(layer).contains(coords.x, coords.y)) {
```

Comment:
It's also possible (for a variety of reasons) that `layer` will be null. If you pass a null layer into `minimumChildBounds` it will throw. I suggest just bailing out early here if `layer` is null.